geoextract/__init__.py

```diff
@@ -45,7 +45,6 @@
 
 from .app import create_app
 
-
```

Comment:
Interesting. I always thought that PEP8 recommends two blank lines after the imports, but I was mistaken. However, I still prefer to have two blank lines there for consistency, since often the first thing after the import section is a function or class definition (which does require two blank lines).